Makefile.inc

```diff
@@ -191,7 +191,10 @@ markdownlint:
 
 # [shellcheck] validates your shell files
 shellcheck:
+# Only run shellcheck if there are files to check
+ifneq (,$(SHELLCHECK_ARGS))
 	shellcheck -x -P $${SCRIPTS_DIR} $(SHELLCHECK_ARGS)
+endif
```

Comment:
Probably should print a warning to stdout clarifying it was skipped because the ARGS is not defined, otherwise it might be confusing

Reply:
Isn’t

    make: Nothing to be done for 'shellcheck'.

clear enough?

Reply:
That's `make` implicit, explicit is better than implicit.

Not everyone is a make guru or understands the intricacies.. Besides if someone did run `make shellcheck` on a project not supporting it I'd expect a clear message such as `Shellcheck is not configured to run on this project` or something like that

Reply:
If we start going down that path, we’ll need to add messages for every similar situation to be consistent. *E.g.*

    $ make bin/subctl
    make: 'bin/subctl' is up to date.

etc. Or am I misunderstanding?

Reply:
I don’t agree that one needs to be a Make guru to understand what “Nothing to be done” means, but whatever, I’ve added a message. I’ve updated the `golangci-lint` patch along similar lines, and I’ll submit other patches to do this on other targets where necessary.

Reply:
> make: Nothing to be done for 'shellcheck'.

This is more clear to me, as I'm accustomed to seeing it from Make and I don't have to think about `if this is incorrect, specify them in SHELLCHECK_ARGS`.

Either way is fine with me.